bbrconn.go

```diff
@@ -11,11 +11,19 @@ import (
 	"github.com/mikioh/tcp"
 )
 
-type InfoCallback func(bytesWritten int, info *tcpinfo.BBRInfo, err error)
+type InfoCallback func(bytesWritten int, info *tcpinfo.Info, bbrInfo *tcpinfo.BBRInfo, err error)
 
 type Conn interface {
 	net.Conn
-	Info() (bytesWritten int, info *tcpinfo.BBRInfo, err error)
+
+	// BytesWritten returns the number of bytes written to this connection
```

Comment:
We don't always need each of these pieces of information, so I broke them into separate functions.